CHANGELOG.md

```diff
@@ -17,5 +19,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - First release.
 
-[Unreleased]: https://github.com/giantswarm/micrologger/compare/v0.1.0...HEAD
+[Unreleased]: https://github.com/giantswarm/micrologger/compare/v0.2.0...HEAD
+[0.2.0]: https://github.com/giantswarm/micrologger/compare/v0.1.0...v0.2.0
```

Comment:
Is there a reason to do the diff here? I think we should be consistent with either the diff view (`/compare...`) or the release details (`/releases...`) for the link destinations. My preference would be release details but your call

Reply:
From the top of this file:

> The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)

I'm just following w/e is there.

Reply:
Ok, I guess my beef is with the "Keep a Changelog" sample then. Carry on